### DISCUSSION TITLE
Enable HTTP/3 for GitHub Pages

### DISCUSSION THREAD

**Discussion by verhovsky** [community/community#28506] [Other Feature Feedback, Questions, & Ideas]
It seems like GitHub Pages uses Fastly. Fastly supports enabling HTTP/3 and I would like to get it

https://docs.fastly.com/en/guides/enabling-http3-for-fastly-services

**Comment by 0x391F** [comment]
I think enable HTTP/3 for ALL GitHub website would be better, especially github.com and github.githubassets.com.
